sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -123,18 +121,18 @@
         new ClosedChannelException(), RntbdClientChannelPool.class, "acquire");
 
     private static final IllegalStateException POOL_CLOSED_ON_ACQUIRE = ThrowableUtil.unknownStackTrace(
-        new StacklessIllegalStateException("service endpoint was closed while acquiring a channel"),
+        new ChannelAcquisitionException("service endpoint was closed while acquiring a channel"),
```

Comment:
do we translate `ChannelAcquisitionException` to GoneException?

Reply:
Yes, it will be translated into GoneException.